README.md

```diff
@@ -73,7 +70,6 @@ Here’s a proposed roadmap for the project. Given the infancy of the project, i
 - [ ] Language X
```

Comment:
Link to [feature request](https://github.com/lyraproj/lyra/issues/new?template=feature_request.md) Issue template! :)

Reply:
Added